spec/v2/index.bs

```diff
@@ -493,14 +493,23 @@ Advisement:
         <td>String
         <td>M
         <td>
-            The IPCC version of the GWP characterization factors used in the calculation of the PCF (see [=Pathfinder Framework=] Section 3.2.2). The value MUST be one of the following:
+          **Deprecated**
+
+          The IPCC version of the GWP characterization factors used in the calculation of the PCF (see [=Pathfinder Framework=] Section 3.2.2). In case several characterization factors are used, indicate the earliest version used in your calculations, disregarding supplier provided PCFs. The value MUST be one of the following:
```

Comment:
We would like to ask the community to help improve the phrasing of the second sentence in the terms of the discussion we had in the Technical Working Group (31/1).